lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -30,15 +30,17 @@ import FileList from './FileList';
 function NoFilesMessage({ href, inSubfolder }) {
   const documentContext = inSubfolder ? 'folder' : 'course';
   return (
-    <div className="FileList__no-files-message">
-      <p>
-        There are no PDFs in this {documentContext}.{' '}
-        <a href={href} target="_blank" rel="noreferrer">
-          Upload some files to the {documentContext}
-        </a>{' '}
-        and try again.
-      </p>
-    </div>
+    <Frame classes="LMSFilePicker__no-files">
+      <div className="u-layout-column--align-center hyp-u-padding--6">
+        <div>
+          There are no PDFs in this {documentContext}.{' '}
+          <a href={href} target="_blank" rel="noreferrer">
+            Upload some files to the {documentContext}
+          </a>{' '}
+          and try again.
+        </div>
+      </div>
+    </Frame>
```

Comment:
Styling this was one of the hardest parts of this bit of work, oddly (per positioning/layout. Long story). A follow-up piece of work is to make the `Table` component potentially accept an optional message to show when it's empty. That would have made this crazy easier.